packages/create-bison-app/template/graphql/modules/user.ts.ejs

```diff
@@ -15,16 +10,13 @@ export const User = objectType({
   name: 'User',
   description: 'A User',
   definition(t) {
-    t.model.id();
-    t.model.email();
-    t.model.roles();
-    t.model.createdAt();
-    t.model.updatedAt();
-    t.model.profile();
+    t.nonNull.id('id');
+    t.nonNull.date('createdAt');
+    t.nonNull.date('updatedAt');
+    t.nonNull.list.nonNull.field('roles', { type: 'Role' });
```

Comment:
This would be equivalent to `[Role!]!` which doesn't allow role to return null or any items within the array to be null. It should allow an empty array though.